Google.Api.Gax/IResourceName.cs

```diff
@@ -13,9 +13,12 @@ namespace Google.Api.Gax
     public interface IResourceName
     {
         /// <summary>
-        /// The kind of resource name.
+        /// Whether this instance contains a known resource name.
         /// </summary>
-        ResourceNameKind Kind { get; }
+        /// <remarks>
+        /// A "known resource name" is one which conforms to one of the pre-defined patterns within this class.
+        /// </remarks>
+        bool IsKnown { get; }
```

Comment:
How about IsKnownPattern? If we have an "unknown" resource name, we still know the resource name - just not the pattern it conforms to.

Reply:
Yes, I agree that's better. I'll change it now.